examples/PhasorDynamics/Tiny/ThreeBus/Classical/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_executable(ThreeBusClassical ThreeBusClassical.cpp)
+target_link_libraries(ThreeBusClassical 
+	GRIDKIT::phasor_dynamics_bus 
+	GRIDKIT::phasor_dynamics_bus_fault 
+	GRIDKIT::phasor_dynamics_branch
+	GRIDKIT::phasor_dynamics_genrou
+	GRIDKIT::phasor_dynamics_gen_classical
+	GRIDKIT::phasor_dynamics_load
+	GRIDKIT::solvers_dyn)
+install(TARGETS ThreeBusClassical RUNTIME DESTINATION bin)
+
+# TODO: add correct answer key and then enable as a test
+# add_test(NAME ThreeBusClassicalTest COMMAND $<TARGET_FILE:ThreeBusClassical>)
```

Comment:
Does this need to be resolved?

Reply:
Yes, it does. We need a reference solution with Powerworld or similar tool to be able to use this example as a test.

Reply:
Addressed in [fe94cd3](https://github.com/ORNL/GridKit/pull/170/commits/fe94cd3cde7775a09fead21f5e21b57e214798a7).